lib/rrule.js

```diff
@@ -2209,13 +2209,40 @@
       }
     },
 
+    /**
+     * Filters out floating timezone properties.
+     * See https://tools.ietf.org/html/rfc5545#section-3.2.19 for more on the time zone property part of the iCal spec.
+     * See https://tools.ietf.org/html/rfc5545#section-3.3.5 for more on floating times in the iCal spec.
+     * @param {String} stringRule A string containing the set of recurrence properties.
+     * @return {String}           A filtered string containing the set of recurrence properties without floating
+     *                            timezones.
+     */
+    filterFloatingTimezones: function (stringRule) {
+      // Uses a regex to split the recurrence string by carriage return.
+      var parsedRecurrenceString = stringRule.split(/\s+/)
```

Comment:
Are we OK with this also matching any spaces? because it will match any spaces too, not just newlines